lib/include/pl/ast/ast_node_attribute.hpp

```diff
@@ -157,14 +157,14 @@ namespace pl {
                 LogConsole::abortEvaluation("pointer base function needs exactly one parameter", node);
 
             if (auto pointerPattern = dynamic_cast<PatternPointer *>(pattern)) {
-                u128 pointerValue = pointerPattern->getPointedAtAddress();
```

Comment:
This will break if you're using a u128 pointer size since it might overflow.

Since it's very unlikely that anybody will ever use u128s as the size though, we might want to just restrict the max size of pointers to an u96/s96

Reply:
I've limited pointer size to max 64 bits, if you don't mind.